### DISCUSSION TITLE
Highlighting only gives back 1 Field

### DISCUSSION THREAD

**Discussion by vsephirothv** [SolrNet/SolrNet#617] [Q&A]
Hey guys, I finished my upload process and my docs are visible on the Solr Instance. 
However, the Highlight (after using a Query either on SolrNet - C# - or using the Solr Admin panel) gives the same results, only shows up 1 field ( in my case ct_id )
Here is the results from my query on Solr AdminPanel 
`{
  "responseHeader":{
    "status":0,
    "QTime":7,
    "params":{
      "df":"sr_cont",
      "hl.bs.type":"WORD",
      "hl":"true",
      "indent":"true",
      "fl":"ct_id, sr_cont, ct_ocr",
      "hl.merge":"false",
      "hl.fl.excluderegex":".*(?:_blob)$",
      "hl.fragsize":"100",
      "q.op":"OR",
      "defType":"lucene",
      "hl.autofields":"true",
      "hl.fl":"*",
      "wt":"json",
      "hl.highlightMultiTerm":"true",
      "facet.field":"ct_cy_0",
      "qt":"standard",
      "hl.snippets":"3",
      "facet.limit":"200",
      "q":"poder",
      "hl.notags":"true",
      "hl.fl.regex":"^(?:(?:sh)|(?:ct))_.+",
      "facet.mincount":"1",
      "hl.method":"unified",
      "facet":"true",
      "facet.sort":"index",
      "useParams":""
    }
  },
  "response":{
    "numFound":1,
    "start":0,
    "numFoundExact":true,
    "docs":[{
      "ct_id":"bf58503f-72be-484a-8df5-f9289e2cde75",
      "ct_ocr":true
    }]
  },
  "facet_counts":{
    "facet_queries":{ },
    "facet_fields":{
      "ct_cy_0":[ ]
    },
    "facet_ranges":{ },
    "facet_intervals":{ },
    "facet_heatmaps":{ }
  },
  "highlighting":{
    "bf58503f-72be-484a-8df5-f9289e2cde75":{ }
  }
}`

What I might possibly be missing here?

Using Solr 9.3.0 if the issue is the version, kindly let me know the latest stable Solr Instance supported version

**Comment by mausch** [comment]
Hi, for general questions about Solr please see https://solr.apache.org/community.html

**Reply by vsephirothv** [reply]
You are kidding right? You reply over 1 year later with a generic response?

**Reply by mausch** [reply]
lol check the dates, my reply was 4 days after your question

**Comment by vsephirothv** [comment]
Why github would notify me only today?

On Mon, Mar 24, 2025 at 7:41 PM Mauricio Scheffer ***@***.***>
wrote:

> lol check the dates, my reply was 4 days after your question
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/SolrNet/SolrNet/discussions/617#discussioncomment-12608161>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABRYBCLWXMKPIBUAJO5KZU32WCCZHAVCNFSM6AAAAABZWBHRBWVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTENRQHAYTMMI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
"They say that pratice leads to perfection. But, since no one is perfect,
why pratice?"
